tests/test_qa.py

```diff
@@ -0,0 +1,21 @@
+import h5py
+import numpy as np
+
+from compass.utils.h5_helpers import QA_PATH
+
+def test_qa_power_stats(geocode_slc_params):
+    def _power_test(arr):
+        return arr > 0.0
+
+    def _phase_test(arr):
+        return abs(arr) > np.pi
```

Comment:
I think I'm not sure how the threshold `np.pi` was determined.

Reply:
[`np.angle`](https://numpy.org/doc/stable/reference/generated/numpy.angle.html) is used to compute phase and it returns (-pi, pi] so nothing should exceed abs(pi).

I will update documentation with explanations.

Reply:
Thanks for the explanation. Then maybe we should switch the inequality like below?
```suggestion
        return abs(arr) <= np.pi
```